words/tails.txt

```diff
@@ -577,3 +577,8 @@ raven
 corvus
 mellori
 bat
+sapsucker
+woodpecker
+nuthatch
```

Comment:
Meta question should we sort them at some point?

Reply:
The chaos is all a part of the fun

Reply:
They are sorted—by time of addition 😆

Reply:
Really, it's largely a timeline of what im thinking and when. This week? I'm with the birds. Next week, maybe I'll be a musical scale 🤷🏻‍♀️I like to live on the edge.